lectures/physics_1/SDLProject/Entity.h

```diff
@@ -1,68 +1,68 @@
 class Entity
 {
 private:
-    int *m_animation_right = NULL, // move to the right
-        *m_animation_left  = NULL, // move to the left
-        *m_animation_up    = NULL, // move upwards
-        *m_animation_down  = NULL; // move downwards
-    
+    int* m_animation_right = NULL, // move to the right
+        * m_animation_left = NULL, // move to the left
+        * m_animation_up   = NULL, // move upwards
+        * m_animation_down = NULL; // move downwards
+
     glm::vec3 m_position;
     glm::vec3 m_velocity;
     glm::vec3 m_acceleration;
-    
-    int m_width  = 1,
+
+    int m_width = 1,
         m_height = 1;
 public:
     // ————— STATIC VARIABLES ————— //
     static const int SECONDS_PER_FRAME = 4;
-    static const int LEFT  = 0,
-                     RIGHT = 1,
-                     UP    = 2,
-                     DOWN  = 3;
-    
+    static const int LEFT = 0,
+        RIGHT = 1,
+        UP    = 2,
+        DOWN  = 3;
+
     // ————— ANIMATION ————— //
-    int **m_walking = new int*[4]
-    {
-        m_animation_left,
-        m_animation_right,
-        m_animation_up,
-        m_animation_down
-    };
-    
+    int** m_walking = new int* [4]
+        {
+            m_animation_left,
+            m_animation_right,
+            m_animation_up,
+            m_animation_down
+        };
+
     int m_animation_frames = 0,
-        m_animation_index  = 0,
-        m_animation_cols   = 0,
-        m_animation_rows   = 0;
-    
-    int  *m_animation_indices = NULL;
-    float m_animation_time    = 0.0f;
-    
+        m_animation_index = 0,
+        m_animation_cols = 0,
+        m_animation_rows = 0;
+
+    int* m_animation_indices = NULL;
+    float m_animation_time = 0.0f;
+
     // ————— TRANSFORMATIONS ————— //
     float     m_speed;
     glm::vec3 m_movement;
```

Comment:
Can you make `m_movement` private here as well? Check out how I have it structures [**here**](https://github.com/sebastianromerocruz/CS3113-material/blob/main/lectures/entities/SDLProject/Entity.h).